File: src/main/java/com/procurement/clarification/model/dto/CreateEnquiryParams.java
package com.procurement.clarification.model.dto;

import java.time.LocalDateTime;
import lombok.Getter;
import lombok.Setter;

@Getter
public class CreateEnquiryParams {
    private final String cpid;
    private final String stage;
    private final String owner;
    private final CreateEnquiryDto dataDto;
    @Setter
    private LocalDateTime date;

    public CreateEnquiryParams(final String cpid,
                               final String stage,
                               final LocalDateTime date,
                               final String owner,
                               final CreateEnquiryDto dataDto) {
        this.cpid = cpid;
        this.stage = stage;
        this.date = date;
        this.owner = owner;
        this.dataDto = dataDto;
    }
}

File: src/main/kotlin/com/procurement/clarification/model/dto/params/CreateEnquiryParams.kt
package com.procurement.clarification.model.dto.params

import com.procurement.clarification.model.dto.request.CreateEnquiryDto
import java.time.LocalDateTime

data class CreateEnquiryParams(
        val cpId: String,
        val stage: String,
        val dateTime: LocalDateTime,
        val data: CreateEnquiryDto
)
